components/merkledb/src/migration.rs

```diff
@@ -547,6 +550,11 @@ pub enum MigrationError {
     /// Migration has been aborted.
     #[fail(display = "Migration was aborted")]
     Aborted,
+
+    /// Never actually generated.
```

Comment:
I'd argue that `MigrationError` here should not admit new variants (at least in MerkleDB 1.x).

Reply:
Then it'll be just inconsistent with the majority of other error types. Are we OK with that?

Reply:
I think yes; users of the main crate are not exposed to this error directly, but rather via `MigrationError` from the `runtime::migrations` module.